docs/troubleshooting.md

```diff
@@ -0,0 +1,105 @@
+# Troubleshooting
+
+## Debugging with the CLI
+
+The easiest way to debug your `release-please` configuration is to use the bundled
+CLI. 
+
+To use the CLI, you will need to use a GitHub personal access token:
```

Comment:
I would link to creating a personal access token:

https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token